File: BookingService/src/main/java/com/booking/dto/Flight.java
package com.booking.dto;
import java.time.LocalDate;
import java.time.LocalTime;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Flight {

    private int flightId;
    private int flightNumber;
    private String flightName;
    private int totalSeats;
    private int availableSeats;
    private String source;
    private String destination;
    private LocalDate flightDate;
    private double duration;
    private int economyClassSeats;
    private int businessClassSeats;    
    private double businessPrice;    
    private double economyPrice;
    private LocalTime takeOff;
    private LocalTime landing;
}

File: src/app/models/Flight.ts
export interface Flight {
  flightId: number;
  flightNumber: number;
  flightName: string;
  totalSeats: number;
  availableSeats: number;
  businessClassSeats: number;
  economyClassSeats: number;
  source: string;
  destination: string;
  takeOff: string;
  landing: string;
  flightDate: string;
  duration: number;
  businessPrice: number;
  economyPrice: number;
}
